### ISSUE TITLE
fonts.lua v2

### ISSUE THREAD

**Issue by fears** [tukui-org/ElvUI#1074]
### What is the issue you are having?

New implementation of how unified fonts should have worked is going mostly awesome :)
-- I don't know if this is actually considered a feature or bug so if I chose unwisely I apologize

Yellow Boxes in the screenshots are where the problem lies

SS1 : Buy Back from vendor (When item stacked) is tiny
![BuyBackVendor](https://github.com/tukui-org/ElvUI/assets/1357322/4f329f88-85ef-41b1-a7f9-6c96c04b94ad)

SS2 : Quest log text is pretty big
![QuestLogText](https://github.com/tukui-org/ElvUI/assets/1357322/7349d329-e6c7-4c05-81a2-ac0a247618a9)

SS3 : Auction house categories are fine, tooltip is fine but the actual items for purchase are large/made larger by modifier
![AhItemsLarge](https://github.com/tukui-org/ElvUI/assets/1357322/6a681584-585d-412e-a53a-ac712456379e)

### What is the expected behavior?

I expect my fonts to be a certain size but theres some kind of modifier making them larger/smaller

### What actually happened?

The fonts are just larger/smaller than expected

### Suggested Solution/Workaround

Change/remove modifiers

### Errors

```shell
No errors. If you feel im being a pain in the ass about this I can just edit the file again lol
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md).
- [X] I have verified ElvUI is up to date by checking /estatus.
- [X] I have verified my issue doesn't exist in the ElvUI Issue Tracker.
- [X] I have not read any of these steps, please close my issue when you see it.
- [X] I have verified my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by kodewdle** [comment]
1) fixed in 638c884b41 it was because `NumberFontNormal` was missing

2) expected behaviour
3) expected behaviour

the fonts in the UI scale similar to how Blizzard scales theirs up.
however ours has a "base size" which can be changed.
the values of the "modifier" are generally closer to the font size selected (smaller range) than blizzards but that behaviour is intended

**Comment by fears** [comment]
I'll just edit the fonts in the lua like I did before the unified fonts it's no big deal. I don't want to come off being ridiculous bothering you I'm sure there's game breaking stuff that's infinitely more important than fonts lol thanks

**Comment by kodewdle** [comment]
optionally, you could also have an external plugin that overwrites or hooks `E:UpdateBlizzardFonts`.  feel free to post any others you find that may have been missed and i can check if they are intentional or not.  will close this one for now as 1 is fixed and the other 2 are intentional.

**Comment by kodewdle** [comment]
(idk if you are on our discord? could've pinged you)  anyways, see: https://github.com/tukui-org/ElvUI/commit/b43720a82c453e4431348e6f405150e25e1bb93e  added a "no font scale" option too ;)

**Comment by fears** [comment]
Yeah I'm on the discord fears/fearswow and I appreciate it man. I know nothing of coding so other than editing text files it would have been a struggle. Back in the day in Pandaria I would edit my own ui but I haven't a clue anymore how I did it so I'm essentially illiterate now lol. I can revert the fonts now and clean up all the ugly green text I wrote to remind myself of what things did in the fonts.lua
